macros/FirefoxOSAPIRef.ejs

```diff
@@ -132,7 +132,7 @@ var badges = {
 }
 
 function buildSublist(pages, title) {
-  var result = '<li data-default-state="open"><a href="#"><strong>' + title + '</strong></a><ol>';
+  var result = '<li class="toggle"><details open><summary>' + title + '</summary><ol>';
```

Comment:
``{{FirefoxOSAPIRef}}`` is deprecated, and also correct 👍.  Screenshot from https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/API/DataStore/get.

<img width="513" alt="pr 789 firefoxosapiref" src="https://user-images.githubusercontent.com/286017/45333295-0ff1c180-b53b-11e8-85d2-e684101e0cfd.png">